benchmarks/time/src/main/scala/strawman/collection/immutable/ImmutableArrayBenchmark.scala

```diff
@@ -88,9 +113,38 @@ class ImmutableArrayBenchmark {
   def map(bh: Blackhole): Unit = bh.consume(xs.map(x => x + 1))
 
   @Benchmark
+  def reverse(bh: Blackhole): Any = bh.consume(xs.reverse)
```

Comment:
Yeah, one benchmark for each of the main abstractions (Seq, IndexedSeq, LinearSeq, Set, SortedSet etc) that could be extended once for each implementation would be super nice. I don't know if this negatively affects the quality of the results though (as the benchmarks will link towards the various abstract classes instead of the implementation class directly; on the other hand that is probably how they would be  used in the wild).